src/server/common/model/answer/kept-separately/kept-separately.js

```diff
@@ -0,0 +1,26 @@
+import { RadioButtonAnswer } from '../radio-button/radio-button.js'
+
+/** @import {RadioButtonConfig} from '../radio-button/radio-button.js' */
+
+/**
+ * export @typedef {'yes'|'no'} KeptSeparatelyData
+ * @typedef {{ keptSeparately: KeptSeparatelyData }} KeptSeparatelyPayload
+ */
+
+/**
+ * @augments {RadioButtonAnswer<KeptSeparatelyPayload>}
+ */
+export class KeptSeparatelyAnswer extends RadioButtonAnswer {
```

Comment:
We have a 'Yes No' model for this.  If you feel that's overkill, then we can remove the model and go ahead with this version.